src/dashmips.ts

```diff
@@ -55,6 +60,10 @@ export class DashmipsDebugClient extends EventEmitter {
         this._readyNotifier.notify()
     }
 
+    private notConnected = (error?: Error) => {
+        setTimeout(() => this.socket.connect(this.port, this.host), 100)
+    }
+
```

Comment:
This function is now the error function until the client connects-- essentially waiting every 0.1 seconds to see if the client is connected.  If it isn't, notConnected is called again.  If it is, then the server proceeds normally by notifying the client.